.gitlab-ci.yml

```diff
@@ -3,7 +3,7 @@
 variables:
   AWS_DEFAULT_REGION: "us-east-1"
 
-  IMAGE_BUILD: cimg/go:1.17-node
+  IMAGE_BUILD: cimg/go:1.18-node
```

Comment:
it is a good practice to pin to a specific digest sha of the image build

Reply:
Is it really in this case? Go only releases security updates as patch releases, so we want to have them every time.

Reply:
Not super critical for this repo but keep in mind that if we pin the dependencies to an exact version in our manifest files, no matter when you build your code, the same top-level dependencies should be pulled in, making our builds more transparent and consistent. This helps a bit in terms of the supply chain security